.gitlab/ci/.gitlab-ci.global.yml

```
@@ -300,19 +300,24 @@
 
 .unlock-machine:
   - section_start "Unlock Machine" --collapsed
-  - source CloudEnvVariables
   - |
-    if [[ -n "${CLOUD_CHOSEN_MACHINE_IDS}" ]]; then
-      if [[ -e "${ARTIFACTS_FOLDER}/logs/lock_file.txt" ]]; then
-        echo "Job finished, removing lock file for machine ids:${CLOUD_CHOSEN_MACHINE_IDS}"
-        gcloud auth activate-service-account --key-file="$GCS_ARTIFACTS_KEY" >> "${ARTIFACTS_FOLDER}/logs/gcloud_auth.log" 2>&1
-        gsutil rm "gs://${TEST_XDR_PREFIX}xsoar-ci-artifacts/$GCS_LOCKS_PATH/machines_locks/*-lock-$CI_JOB_ID"
-        echo "Finished removing lock file"
+    if [[ -f "CloudEnvVariables" ]]; then
+      source CloudEnvVariables
+
+      if [[ -n "${CLOUD_CHOSEN_MACHINE_IDS}" ]]; then
+        if [[ -e "${ARTIFACTS_FOLDER}/logs/lock_file.txt" ]]; then
+          echo "Job finished, removing lock file for machine ids:${CLOUD_CHOSEN_MACHINE_IDS}"
+          gcloud auth activate-service-account --key-file="$GCS_ARTIFACTS_KEY" >> "${ARTIFACTS_FOLDER}/logs/gcloud_auth.log" 2>&1
+          gsutil rm "gs://${TEST_XDR_PREFIX}xsoar-ci-artifacts/$GCS_LOCKS_PATH/machines_locks/*-lock-$CI_JOB_ID"
+          echo "Finished removing lock file"
```

There could be several, right?
Also, can we run the `gsutil rm` with `verbose` or something?
```suggestion
          echo "Finished removing lock file(s)"
```